src/Lykke.Service.ClientSearch/Startup.cs

```diff
@@ -116,8 +120,18 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
                 // NOTE: Service not yet recieve and process requests here
 
                 await ApplicationContainer.Resolve<IStartupManager>().StartAsync();
-
                 await Log.WriteMonitorAsync("", $"Env: {Program.EnvInfo}", "Started");
+
+                Indexer indexer = ApplicationContainer.Resolve<Indexer>();
+                await Task.Factory.StartNew(
+                    async () => {
+                        await indexer.Initialize();
+
+                        _triggerHost = new TriggerHost(new AutofacServiceProvider(ApplicationContainer));
+                        _triggerHostTask = _triggerHost.Start();
```

Comment:
_triggerHostTask & _triggerHost  are used only here. Why they are fields?

Reply:
moved in to new IndexerManager class